src/features/customerView/invoiceModal/InvoiceModal.tsx

```diff
@@ -6,105 +6,167 @@ import { isBerthInvoice, isWinterStorageInvoice } from '../utils';
-          <LabelValuePair
-            label={t('common.terminology.berth')}
-            value={
-              invoice.berthInformation.harborName +
-              ' ' +
-              invoice.berthInformation.pierIdentifier +
-              ' ' +
-              invoice.berthInformation.number
-            }
-          />
-        )}
-        {isWinterStorageInvoice(invoice) && (
-          <LabelValuePair
-            label={t('common.terminology.winterStorageArea')}
-            value={invoice.winterStorageInformation.winterStorageAreaName}
-          />
-        )}
-        <LabelValuePair
-          label={t('customerView.customerInvoice.contractPeriod')}
-          value={`${formatDate(contractPeriod.startDate, i18n.language)} - ${formatDate(
-            contractPeriod.endDate,
-            i18n.language
-          )}`}
-        />
-        <LabelValuePair
-          label={t('customerView.customerInvoice.dueDate')}
-          value={formatDate(invoice.dueDate, i18n.language)}
-        />
-        {invoice.status === OrderStatus.PAID && (
-          <LabelValuePair
-            label={t('customerView.customerInvoice.paidAt')}
-            value={formatDate(invoice.paidAt, i18n.language)}
+    <Modal className={styles.invoiceModal} toggleModal={() => toggleModal?.(false)} {...modalProps}>
+      {invoice && (
+        <>
+          <StatusLabel
+            className={styles.statusLabel}
+            type={ORDER_STATUS[invoice.status].type}
+            label={t(ORDER_STATUS[invoice.status].label)}
           />
-        )}
-        <LabelValuePair
-          label={t('customerView.customerInvoice.status')}
-          value={t(getOrderStatusTKey(invoice.status))}
-        />
-      </Section>
 
-      <hr className={styles.divider} />
+          <div className={styles.heading}>
+            <Text as="h4" color="brand">
+              {t('common.terminology.invoice').toUpperCase()}
+            </Text>
 
-      <Section>
-        <LabelValuePair
-          label={t('common.terminology.basePrice')}
-          value={formatPrice(invoice.basePrice, i18n.language)}
-        />
-        {invoice.orderLines.map((orderLine, id) => (
-          <LabelValuePair
-            label={t(getProductServiceTKey(orderLine.product))}
-            value={
-              orderLine.priceUnit === PriceUnits.PERCENTAGE
-                ? formatPrice(orderLine.price, i18n.language, orderLine.priceValue)
-                : formatPrice(orderLine.price, i18n.language)
-            }
-            key={id}
-          />
-        ))}
-      </Section>
+            <InvoiceActions
+              className={styles.select}
+              selectedAction={selectedAction}
+              actions={[
+                {
+                  value: actions.sendOffer.value,
+                  label: actions.sendOffer.label,
+                  disabled: actions.sendOffer.disabled,
+                  onClick: actions.sendOffer.onSelect,
+                },
```

Comment:
I think we could shrink these lines down somehow, but that's not strictly required

Reply:
Done! Now, these are slightly shorter.